src/index.js

```diff
@@ -0,0 +1,195 @@
+const cityUpdate = () => {
+  const newCityText = state.cityTextField.value;
+  state.headerCity.textContent = newCityText;
+};
+
+const cityDefault = "Boston";
+
+const cityReset = () => {
+  state.cityTextField.value = cityDefault;
+  state.headerCity.textContent = state.cityTextField.value;
+};
+
+const landscapes = {
```

Comment:
Objects (dicts) are good at looking up specific values, but in this case, a range of values need to map to each of the landscapes. Since we are going to need to check each of the ranges anyway, we could instead us a list, and store information about the temperature ranges each list entry applies to. If we are careful about the order we store them, we could store either just the top or bottom end of the range as follows:
```js
const landscapes = [
    [80, "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂", "red"],
    [70, "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷", "orange"],
    [60, "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃", "green"],
    [50, "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲", "blue"],
];
```

Notice that we could also include the class name to apply to the temperature as well! When the temperature is set, we could iterate through the list to find the first record less than or equal to the current temperature. So if the temp was 100, we would match the 80 record. Whereas if the temp was 65, we would match the 60 record. If we find no matching record, we could take the final one.

From the matching record, we can determine both the landscape to use, as well as the class to apply to the temperature display, and could avoid brittle chains of `if`/`else if` checks. This also lends itself well to storing the temperature breaks in a database, allowing us to update the UI behavior without modifying code!

In the current implementation, there's a difference in the temperature breaks for the landscape and the colors. We could decide to eliminate the differences, store multiple lists, or include additional logic to handle records that effect one or the other (this would be the most complex).